documentation/src/main/markdown/configuring/commandline.md

```diff
@@ -12,5 +12,5 @@ There is a space before and between each complete property setting, but there ar
 For example,
 to set property *detect.project.name*:
 ```
-bash <(curl -s -L https://detect.synopsys.com/detect9.sh) --detect.project.name=MyProject
+bash <(curl -s -L https://detect.blackduck.com/detect9.sh) --detect.project.name=MyProject
```

Comment:
If these changes are going in to 10.0.z branch, feel free to update to detect10.sh